docsrc/imap/developer/compiling.rst

```diff
@@ -178,6 +178,10 @@ Other
     installing SSL certificates."
     `ClamAV`_, libclamav-dev, clamav-devel, "no", "It is used by
     **cyr_virusscan**, otherwise not needed."
+    `CLD2`_, libcld2-dev, cld2-devel, "yes/no", "Compact Language Detector 2
+    (probabilistically detects over 80 languages in Unicode UTF-8 text, either
+    plain text or HTML/XML). Required for **Xapian** (``--enable-xapian``),
```

Comment:
It is required for Xapian if the search_index_language option in imapd.conf is set. Thanks!

Reply:
I just added the clarification about `search_index_language`.

Reply:
@rsto What happens if `search_index_language` is enabled on an installation that was compiled without CLD2?

Reply:
I guess we get the error @brong mentioned in the related issue: "IOERROR: Xapian: language indexing requires CLD2 library"